src/style.css

```
@@ -0,0 +1,42 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+  --light-star: url("./images/star-active.svg");
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.star {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.star:nth-child(n+2) .stars__star:nth-child(1) {
+  background-image: var(--light-star);
+}
+
+.star:nth-child(n+3) .stars__star:nth-child(2) {
+  background-image: var(--light-star);
+}
```

Let's merge this styles)

```suggestion
.star:nth-child(n+2) .stars__star:nth-child(1),
star:nth-child(n+3) .stars__star:nth-child(2) {
  background-image: var(--light-star);
}
```